hbase-server/src/main/java/org/apache/hadoop/hbase/master/HMaster.java

```diff
@@ -2763,6 +2763,13 @@ protected String getDescription() {
   private long modifyTable(final TableName tableName,
     final TableDescriptorGetter newDescriptorGetter, final long nonceGroup, final long nonce,
     final boolean shouldCheckDescriptor) throws IOException {
+    return modifyTable(tableName, newDescriptorGetter, nonceGroup, nonce, shouldCheckDescriptor,
```

Comment:
I was looking into why we need this overload. I see this is called from addColumn/deleteColumn/modifyColumn. Does it make sense to add reopenRegion support to modifyColumn at least?

Reply:
Now we have two methods:

`    modifyTable(..[five parameters]..)`
`    modifyTable(..[five parameters].., boolean reopenRegions)`

The methods 'addColumn/deleteColumn/modifyColumn' that you mentioned will call method `modifyTable(..[five parameters]..)` directly as before. They are not aware of the 'reopenRegions' parameter.

To make the code structure cleaner, I only changed one line in method `modifyTable(..[five parameters]..)`, which then turns into `modifyTable(..[five parameters].., boolean reopenRegions)`. So now, you can see that `modifyTable(..[five parameters]..)` calls `modifyTable(..[five parameters].., boolean reopenRegions)` with `reopenRegions=true`.